src/app/app.component.ts

```diff
@@ -312,6 +314,14 @@ export class AppComponent implements OnInit {
                         break;
                     case 'sharingPaywall':
                         this.isSharingPaywallDisplay = true;
+                        break;
+                    case 'navigate':
```

Comment:
What are the pros to use the messaging service to navigate instead of using the `router` injection from the caller component?

Reply:
Fixed later. No reason.